sdk/keyvault/azure-security-keyvault-secrets/README.md

```diff
@@ -344,7 +344,7 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 <!-- LINKS -->
 [source_code]:  src
 [api_documentation]: https://azure.github.io/azure-sdk-for-java
-[azure_identity]: https://github.com/Azure/azure-sdk-for-java/tree/master/identity/client
+[azure_identity]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/identity/azure-identity
```

Comment:
Do we have the wrong link in the GA release README for Keys and Secrets? If so, can you check with EngSys if it can be rectified?

Reply:
yeah, will ask them.